### DISCUSSION TITLE
GRPC调用失败

### DISCUSSION THREAD

**Discussion by portalll** [secretflow/secretflow#863] [Q&A]
panic: rpc error: code = Unimplemented desc = unknown service proto.JobService [recovered]
	panic: rpc error: code = Unimplemented desc = unknown service proto.JobService。本地容器中查询不到相关grpc服务，http端口正常，请问有GRPC调用样例吗

**Comment by zhouaihui** [comment]
hi, @portalll , 麻烦去对应repo下提交issue。
